src/subcommands/publishers.rs

```diff
@@ -48,6 +63,12 @@ pub fn publishers(args: Vec<String>, max_age: std::time::Duration) -> Result<(),
             }
         }
         println!("\nGithub teams are black boxes. It's impossible to get the member list without explicit permission.");
+    } else if diffable {
+        let sorted_map = sort_transposed_map_for_diffing(team_to_crate_map);
+        for (team, crates) in sorted_map.iter() {
+            let crate_list = comma_separated_list(&crates);
+            println!("team \"{}\": {}", &team.login, crate_list);
+        }
```

Comment:
This condition is confusing. The `if` is not entered if `diffable` so it seems to be simpler to follow this code if we make this the first branch, which simplifies the non-diffable branch's condition to `!publisher_teams.is_empty()` again.

Reply:
Good catch! Thanks!